evap/grades/views.py

```diff
@@ -127,6 +128,7 @@ def toggle_no_grades(request):
 
     course.gets_no_grade_documents = not course.gets_no_grade_documents
     course.save()
+    # specific ordering?
```

Comment:
ordering would only be shown in the emails, we have another pr working on this (will probably done in the templates).